xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -405,8 +510,22 @@
         <ReturnType>System.Func&lt;System.Runtime.Loader.AssemblyLoadContext,System.Reflection.AssemblyName,System.Reflection.Assembly&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Occurs when the resolution of an assembly fails.</summary>
```

Comment:
I would explicitly state that this fires when an assembly resolution **on this ALC** fails - since the event is per-context.